pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -1201,6 +1201,10 @@ type KubevirtCloudSpec struct {
 	// InfraStorageClasses is a list of storage classes from KubeVirt infra cluster that are used for
 	// initialization of user cluster storage classes by the CSI driver kubevirt (hot pluggable disks)
 	InfraStorageClasses []string `json:"infraStorageClasses,omitempty"`
```

Comment:
I will deprecate `InfraStroageClasses` in the next PR where the deprecation will be handed properly.
If I deprecate now, we have linter issues and then would need to handle everything is this PR. But the goal here is only to add the fields to ease testing.